src/LinkedIn.php

```diff
@@ -277,7 +277,7 @@ protected function getUserFromAccessToken()
         try {
             return $this->api('GET', '/v1/people/~:(id,firstName,lastName,headline)');
         } catch (LinkedInApiException $e) {
-            return;
+            return null;
```

Comment:
It is a part of the [PSR fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) and the Symfony coding standards. 

Please erase them (or run PSR fixer and let it fix it for you)